File: extract_spec.py
import json
import os

log_path = r'C:\Users\durrm\.gemini\antigravity\brain\71706f8a-762b-4900-aadc-45d945f42fa7\.system_generated\logs\overview.txt'
with open(log_path, 'r', encoding='utf-8') as f:
    first_line = f.readline()
    data = json.loads(first_line)
    content = data.get('content', '')
    with open('full_spec.md', 'w', encoding='utf-8') as out:
        out.write(content)

File: extract_spec.js
const fs = require('fs');
const logPath = 'C:\\Users\\durrm\\.gemini\\antigravity\\brain\\71706f8a-762b-4900-aadc-45d945f42fa7\\.system_generated\\logs\\overview.txt';
const firstLine = fs.readFileSync(logPath, 'utf8').split('\n')[0];
const data = JSON.parse(firstLine);
fs.writeFileSync('full_spec.md', data.content);
console.log('Extracted full_spec.md');
